lib/puppet/provider/network_snmp/nxapi.rb

```diff
@@ -26,34 +26,34 @@
 end
 
 Puppet::Type.type(:network_snmp).provide(:nxapi) do
-  desc "The Cisco NXAPI provider for network_snmp."
+  desc 'The Cisco NXAPI provider for network_snmp.'
 
-  confine :feature => :cisco_node_utils
-  defaultfor :operatingsystem => :nexus
+  confine feature: :cisco_node_utils
+  defaultfor operatingsystem: :nexus
 
   mk_resource_methods
 
-  NETWORK_SNMP_PROPS={
-    :enable => :protocol,
-    :contact => :contact,
-    :location => :location,
+  NETWORK_SNMP_PROPS = {
+    enable:   :protocol,
+    contact:  :contact,
+    location: :location,
   }
 
   def initialize(value={})
     super(value)
-    @network_snmp = Cisco::SnmpServer.new()
+    @network_snmp = Cisco::SnmpServer.new
     @property_flush = {}
-    debug "Created provider instance of network_snmp"
+    debug 'Created provider instance of network_snmp'
   end
 
-  def self.get_properties
-    network_snmp = Cisco::SnmpServer.new()
+  def self.get_properties # rubocop:disable Style/AccessorMethodName
```

Comment:
Right, we decided on property_get and property_set a while back.  Submitted new diff